plugins/Widgetize/lang/en.json

```
@@ -4,6 +4,19 @@
         "PluginDescription": "Display any Matomo report in your website or app with a simple \"embed\" HTML tag.",
         "ViewAccessRequired": "This user has at least some write access. Only tokens of users who have only view access can be used. Check %1$s for more info.",
         "TopLinkTooltip": "Export Matomo reports as widgets and embed the dashboard in your app as an iframe.",
-        "TooHighAccessLevel": "This user has super-user access. Embedding widgets with super-user token authentication is not allowed. %1$sCheck the FAQ for more info.%2$s"
+        "TooHighAccessLevel": "This user has super-user access. Embedding widgets with super-user token authentication is not allowed. %1$sCheck the FAQ for more info.%2$s",
+        "SelectAReport": "Select a report, and copy paste in your page the embed code below the widget:",
+        "Reports": "Widgetize reports",
+        "Intro": "With Matomo, you can %s on your blog, website, or intranet dashboard... in one click.",
+        "Intro2": "export your Web Analytics reports",
```

Could yo combine them again and only insert the html tags with placeholders? Otherwise it's hard for the translators to know what the placeholder will be filled with, which might lead to bad translations...
```suggestion
        "Intro": "With Matomo, you can %1$sexport your Web Analytics reports%2$s on your blog, website, or intranet dashboard… in one click.",
```